apps/docs/docs/reference/ui/molecules/ProductCard.mdx

```diff
@@ -0,0 +1,181 @@
+          brand: {
+            brandName: 'Brand',
+            name: 'Brand'
+          },
+          gtin: '5595633577807',
+          id: '15503951',
+          image: [
+            {
+              alternateName: 'est',
+              url: 'http://storeframework.vtexassets.com/arquivos/ids/190191/numquam.jpg?v=637755599170100000'
+            }
+          ],
+          isVariantOf: {
+            name: 'Handmade Steel Towels Practical',
+            productGroupID: '130742'
+          },
+          name: 'red',
+          offers: {
+            lowPrice: 181.71,
+            offers: [
+              {
+                availability: 'https://schema.org/InStock',
+                listPrice: 208.72,
+                price: 181.71,
+                quantity: 1,
+                seller: {
+                  identifier: '1'
+                }
+              }
+            ]
+          },
+          sku: '15503951',
+          slug: 'handmade-steel-towels-practical-15503951'
+        }}
+        variant="default"
+      />
+    </div>
+  </Provider$2>
+</LocationProvider>
+```
+
+### Wide
+```tsx live
+<LocationProvider>
+  <Provider$2
+    initialState={{
+      channel: '{"salesChannel":"1"}'
+    }}
+  >
+    <div
+      style={{
+        width: 400
+      }}
+    >
+      <ProductCard
+        ButtonBuy={null}
+        aspectRatio={1.5}
+        index={1}
+        product={{
+          brand: {
+            brandName: 'Brand',
+            name: 'Brand'
+          },
+          gtin: '5595633577807',
+          id: '15503951',
+          image: [
+            {
+              alternateName: 'est',
+              url: 'http://storeframework.vtexassets.com/arquivos/ids/190191/numquam.jpg?v=637755599170100000'
+            }
+          ],
+          isVariantOf: {
+            name: 'Handmade Steel Towels Practical',
+            productGroupID: '130742'
+          },
+          name: 'red',
+          offers: {
+            lowPrice: 181.71,
+            offers: [
+              {
+                availability: 'https://schema.org/InStock',
+                listPrice: 208.72,
+                price: 181.71,
+                quantity: 1,
+                seller: {
+                  identifier: '1'
+                }
+              }
+            ]
+          },
+          sku: '15503951',
+          slug: 'handmade-steel-towels-practical-15503951'
+        }}
+        variant="wide"
+      />
+    </div>
+  </Provider$2>
+</LocationProvider>
+```
+### Use cases
+Use the `ProductCard` in the following context:
+- Product Collections on the home page.
+- Product availability statuses on the home pages, e.g., *Just Arrived*.
+- Carousel with product promotions.
+
+## Props
+
+<PropsSection name="ProductCard" />
+
+## Customization
+`data-store-product-card`
+`data-product-card-content`
+`data-product-card-actions`
+`data-store-price`
+`data-store-badge`
+`data-store-button`
+
+
+## Best practices
+
+### Do's
+- Use high-quality images.
```

Comment:
high-quality images also means a good image of the product that can attract more customers? 🤔